django-backend/fecfiler/transactions/schedule_b/managers.py

```diff
@@ -28,6 +28,7 @@
     "NATIONAL_PARTY_RECOUNT_ACCOUNT_DISBURSEMENT",
     "NATIONAL_PARTY_HEADQUARTERS_ACCOUNT_DISBURSEMENT",
     "NATIONAL_PARTY_CONVENTION_ACCOUNT_DISBURSEMENT",
+    "IN_KIND_OUT",
```

Comment:
IN_KIND_OUT is always itemized regardless of amount according to the ticket, so it should not be in this list.

Reply:
This requirement changed recently and the acceptance criteria is to auto itemize only  for > $200.01